rbd/rbd_options_test.go

```diff
@@ -0,0 +1,156 @@
+package rbd_test
+
+import (
+	"github.com/ceph/go-ceph/rbd"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestRbdOptions(t *testing.T) {
+	var i uint64
+	var s string
+	var err error
+
+	options := rbd.NewRbdImageOptions()
+
+	err = options.SetUint64(rbd.RbdImageOptionFormat, 1)
```

Comment:
<did not read fully the code>
Are you setting the Image Format to v1 here?
</did not read fully the code>

If you do, it should be v2.
If I misread - please ignore this comment ;-)

Reply:
@mykaul This is "just" unit test code so it's fair to exercise any valid options the API supports. That said, if option format v2 give vastly different behavior (for the api) compared to v1 we should test both code paths. If not, this is probably fine.